inabox/tests/test_harness.go

```diff
@@ -32,8 +33,9 @@ type InfrastructureHarness struct {
 
 	// EigenDA components
 	// TODO: Should EigenDA components be their own test harness?
```

Comment:
yes they should imo. can be embedded in here if we so decide but imo should have their own namespace.

Reply:
hmm actually now come to think of it chain is shared but graphnode is eigenda disperser instance specific right?

Reply:
maybe worth a live chat